config/initializers/letter_opener.rb

```diff
@@ -0,0 +1,5 @@
+if Object.const_defined? :LetterOpener
+  LetterOpener.configure do |config|
+    config.file_uri_scheme = "file://"
```

Comment:
Default, for me, was `mailto:` which gave focus to the browser but didn't work and was annoying.